app/src/main/res/layout/testing_inapp.xml

```diff
@@ -0,0 +1,146 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@benmarten or @Enuviel Can you share your settings for xml formatting for android studio? I just formatted this with the default

Reply:
I think default is totally fine ;)